File: Python3/프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    a_str, operation, b_str = binomial.split(" ")
    
    a, b = int(a_str), int(b_str)
    
    if operation == "+":
        return a + b
    elif operation == "-":
        return a - b
    else:
        return a * b

File: Swift/프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.swift
import Foundation

func solution(_ binomial:String) -> Int {
    let parts = binomial.split(separator: " ").map{ String($0) }
    
    if let a = Int(parts[0]), let b = Int(parts[2]) {
        let operation = parts[1]
        
        switch operation {
            case "+": return a + b 
            case "*": return a * b
            case "-": return a - b
            default: return 0
        }
    }
    return 0
}